src/main/java/apollo/upload/MavenUploader.java

```diff
@@ -19,6 +21,8 @@
     private final MavenDeployer mavenDeployer;
     private final XmlReformer xmlReformer;
     private static final Logger logger = LogManager.getLogger(MavenUploader.class);
+    private static final int maxThreadNo = 8;
```

Comment:
consider to use more sophisticated way to determine the number of threads instead of constant number

Reply:
Done